src/common/com/intellij/plugins/haxe/util/HaxeResolveUtil.java

```diff
@@ -526,8 +526,13 @@ public static HaxeClass tryResolveClassByQName(@Nullable PsiElement type) {
       return null;
     }
 
+    // can be accessed without full QName
+    HaxeClass result = findClassByQName(type.getText(), type.getContext());
```

Comment:
Unfortunately i've not add unit-test for this because I forgot to do it. The point is - `getQName(test)` broke name if it's already in QName format, so after that resolve failed. Let me know if I should add tests for exactly this case. Thank you :)